cli/src/pcluster/resources/login_node/user_data.sh

```diff
@@ -114,7 +114,13 @@ write_files:
         # wait logs flush before signaling the failure
         sleep 10
         # TODO: add possibility to override this behavior and keep the instance for debugging
-        shutdown -h now
+        # Notify the AutoScalingGroup to proceed with adding the instance for service
+        IMDS_TOKEN=$(curl -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 300")
+        INSTANCE_ID=$(curl -H "X-aws-ec2-metadata-token: $IMDS_TOKEN" -v http://169.254.169.254/latest/meta-data/instance-id)
+        aws autoscaling complete-lifecycle-action --auto-scaling-group-name "${AutoScalingGroupName}" --lifecycle-hook-name "${LaunchingLifecycleHookName}" --instance-id "$INSTANCE_ID" --lifecycle-action-result CONTINUE --region "${AWS::Region}"
+
+        # Notify CloudFormation that the bootstrap process failed
+        cfn-signal --exit-code=1 --reason="Failed to launch login node" --region ${AWS::Region}  --url ${CloudFormationUrl} "${WaitConditionURL}"
```

Comment:
What about adding the instance id too in the reason? I mean something like: "Failed to launch login node $INSTANCE_ID"

Reply:
CF already returns the Id ... in case of error the message we receive is 

```
WaitCondition received failed message: 'Failed to launch login node' for uniqueId: i-00b68fd8f023eb9de
```